corehq/blobs/migrate.py

```diff
@@ -517,7 +517,7 @@ def __init__(self, slug, doc_types, doc_migrator_class):
         sorted_types = sorted(doc_type_tuples_to_dict(self.doc_types))
         self.iteration_key = "{}-blob-migration/{}".format(self.slug, " ".join(sorted_types))
 
-    def migrate(self, filename=None, reset=False, max_retry=2, chunk_size=100):
+    def migrate(self, filename=None, reset=False, max_retry=2, chunk_size=100, **options):
```

Comment:
would be nice if any migrator that accepts options but does not use them would raise an exception if options are passed. That way invalid/unexpected options are not silently ignored.

Better yet, add named arguments to any migrator that accepts extra options, and not to ones that do not.